README.md

```diff
@@ -9,18 +9,23 @@ Based on https://github.com/joshdvir/drone-ecs-deploy and https://github.com/sil
 ## Usage
 
 You will need to set the `cluster`, `image`, `region` and `role` properties in
-your drone config and additionally the `services` (to update ECS services) and
-`tasks` (for running one-off tasks) properties.
+your drone config and additionally the `services` (to update ECS services),
+`tasks` (for running one-off tasks) and `commands` (for executing commands
+before deployment) properties.
 
 Each entry in `services` will change the service with the same name, changing
 the image in the task definition and update the service with the new definition.
 
 If `tasks` is set, it's also required to provide the `task_definition` property,
-as it can't be determined automatically (like with services).
-
-Each line is equivalent to one executed task and `CMD` (in Docker) will be set
-to this value. It's not guaranteed, that the tasks will execute in order.
+as it can't be determined automatically (like with services). Each line is
+equivalent to one executed task and `CMD` (in Docker) will be set to this
+value. It's not guaranteed, that the tasks will execute in order, and they will
+run simultaneously with the updating of the services.
 
+If `commands` are set it's also required to provide an `exec_service` where they
```

Comment:
I think we should give this a more precise name, so that its clear when looking at it that it just executes pre deployment commands in the old container. Right now it seems to behave like the `command` property from docker compose to pass additional arguments. Maybe something like `pre_commands`?

Reply:
I chose `exec_commands` to be consistent with `exec_service`, see https://github.com/formunauts/drone-ecs-deploy-advanced/pull/1/commits/f208cb775af3a7527010b7d3f4626a1b4f019b71.